tests/phpt/dl/483_bcmath.php

```diff
@@ -160,4 +160,73 @@ function test_bcmath_negative_zero() {
   #endif
 }
 
+function test_bcsqrt($nums) {
+  foreach ($nums as $num) {
+    echo bcsqrt($num), "\n";
+
+    for ($i = 0; $i < 30; ++$i) {
+      echo "num: ", $num, "; scale: ", $i, "; sqrt: ", bcsqrt($num, $i), "\n";
```

Comment:
nit: php variables in double quoted strings are substituted with their values
```suggestion
      echo "num: $num; scale: $i; sqrt: ", bcsqrt($num, $i), "\n";
```